### PR TITLE
AutomatticBylineLogo | JetpackLogo: Changing `title` tag to `desc`

### PR DESCRIPTION
**Pull request by renatoagds**
Using `title` inside `svg` go against [W3C](https://www.w3.org/TR/html401/struct/global.html#h-7.4.2).

#### Changes proposed in this Pull Request:

* Changing `title` for `desc`.

#### Other information:

- [x] Have you written new tests for your changes, if applicable?
- [x] Have you checked the E2E test CI results, and verified that your changes do not break them?

#### Jetpack product discussion

N/A

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* Start `Storybook`
* Go to `AutomatticBylineLogo` story.
* You should see `desc` tag instead `title`.

### CODE REVIEW

**Review by manzoorwanijk — DISMISSED** [review]
Thank you for the quick fix.

**Review by oskosk — DISMISSED** [review]
thanks!

File: projects/js-packages/components/changelog/fix-inline-logo-description [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fixed
+
+AutomatticBylineLogo: Change title by desc
```
Comment (sdixon194) [inline projects/js-packages/components/changelog/fix-inline-logo-description]:
```suggestion
AutomatticBylineLogo: Change title tag to desc
```

**Review by sdixon194 — DISMISSED** [review]

File: projects/js-packages/components/changelog/fix-inline-logo-description (hunk 1 above)
Comment (renatoagds) [reply projects/js-packages/components/changelog/fix-inline-logo-description]: @sdixon194 thanks!

**Review by sdixon194 — APPROVED** [review]

**Comment by renatoagds** [comment]
r248343-wpcom

**Pull request merged**